Makefile

```diff
@@ -18,12 +18,15 @@ clean:
   		echo "Nothing to clean."; \
 	fi
 
+# Installs without rebuilding apps.
+install: install-safe install-sidecar install-sentinel
+
+# Installs the demo app to play with.
+demo: install-demo
```

Comment:
Better to split installation of the demo workload since it is not part of the production setup.

It also make sense to install SPIRE as a separate step since the user may already have SPIRE rolled out.